geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -18,6 +18,7 @@
 from glob import glob
```

Comment:
Leaving a conversation open to remind us to NOT merge this PR until the PR is pionting to main (which will automatically happen once v1.12.0-release is merged to main).